keyboards/acheron/austin/keymaps/WK/keymap.c

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This keymap seems to be identical to the default. Please delete it and rename the `WKL` keymap to lowercase.

Reply:
Renamed both WKL and WK to lowercase